lib/longshoreman/container.rb

```diff
@@ -76,7 +74,7 @@ def rport(exposed_port, protocol="tcp")
   end
 
   def start
-    @raw.start
+    @raw.start({ 'PublishAllPorts' => true, 'CapAdd' => 'NET_ADMIN' })
```

Comment:
for my zookeeper container/image I was receiving this as the network settings before this fix

```
"Ports"=>{"2181/tcp"=>nil, "2888/tcp"=>nil, "3888/tcp"=>nil}
```

after fix

```
{"2181/tcp"=>[{"HostIp"=>"0.0.0.0", "HostPort"=>"49156"}], "2888/tcp"=>[{"HostIp"=>"0.0.0.0", "HostPort"=>"49157"}], "3888/tcp"=>[{"HostIp"=>"0.0.0.0", "HostPort"=>"49158"}]}
```